pkgs/os-specific/linux/tuxedo-control-center/update.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
Please use nix-shell with curl and node2nix.

Reply:
Done.